crates/anvil/src/anvil.rs

```diff
@@ -56,6 +73,19 @@ async fn main() -> eyre::Result<()> {
                 "anvil",
                 &mut std::io::stdout(),
             ),
+            AnvilSubcommand::Reorg { depth, new_len, transactions_path, rpc_url } => {
+                let url = rpc_url.clone().unwrap_or("127.0.0.1:8545".to_string());
+                let provider = get_http_provider(url);
+
+                let tx_block_pairs = if let Some(path) = transactions_path {
+                    read_json_file::<Vec<(TransactionRequest, u64)>>(path.as_ref())?
+                } else {
+                    Vec::new()
+                };
+
+                let params = serde_json::json!([*depth, *new_len, tx_block_pairs]);
+                provider.raw_request("anvil_reorg".into(), params).await?;
+            }
```

Comment:
hmm, unsure we really want this as additional command because all this does is sending a rpc request, so I'd like to not include this

Reply:
Yep, good call out. I was also tossing up on this a bit too. But I thought this was the most ergonomic way to supply a file path to a list of transactions to mine into the new blocks.

Will remove.

Reply:
@mattsse could I please ask for a suggestion on how the user would supply the transactions to be mined into the new blocks via an RPC request? I felt this was a cumbersome and potentially a bad DX for the user. Thank you.